### PR TITLE
meta: update setup.py for pypi

### PR DESCRIPTION
**Pull request by ryzokuken**
Now we have https://pypi.org/project/gyp-next/0.1/

/cc @targos @cclauss

### CODE REVIEW

**Comment by ryzokuken** [comment]
@cclauss looks better? Will update the actual page once this is merged.

**Comment by cclauss** [comment]
Let’s add a `python_requires` like https://github.com/psf/requests/blob/master/setup.py#L82

We also need PyPI trove classifiers https://pypi.org/pypi?%3Aaction=list_classifiers at a minimum for the Python versions that we support and `License :: OSI Approved :: BSD License`

https://github.com/nodejs/tap2junit/blob/master/setup.py#L18 feels right to me but at a minimum you should at least add yourself to the `AUTHORS` file.

**Comment by targos** [comment]
@cclauss PTAL. We would like to tag a first release (without publishing to PyPI) and downstream back to node and node-gyp.

File: setup.py [hunk 1]
```diff
@@ -4,16 +4,41 @@
...
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```
Comment (cclauss) [inline setup.py:37]: Do we really need these two lines?  ;-)

Comment (ryzokuken) [reply setup.py:37]: I copied them, but I have no clue. Feel free to remove anything you deem unnecessary in a PR, I'd +1 that.

**Comment by ryzokuken** [comment]
(merging for now, thanks everyone)

**Pull request merged**